### DISCUSSION TITLE
Ninja 2.7.12

### DISCUSSION THREAD

**Discussion by szapp** [szapp/Ninja#27] [General]
Support the [latest changes of Ikarus](https://github.com/Lehona/Ikarus/compare/93591080ecbb7880dd7808adb0b88c833421a77c...bae41b582b568780193691f57564add6ba5b18ee) (i.e. hotfix for latest change in version 2.7.11).

- [Change log](../../wiki/Changelog#v2712-sep-5-2021)
- [Installation instructions](../../wiki/Installation-(EN)#wiki-wrapper)
- [Check sums](../../wiki/Checksums#wiki-wrapper)

<hr /><em>This discussion was created from the release <a href='https://github.com/szapp/Ninja/releases/tag/v2.7.12'>Ninja 2.7.12</a>.</em>

**Comment by justtoask** [comment]
How to install it? I own the game on steam and already installed it trough spine with union and community patch, but simply ticking it doesn't work.

**Reply by szapp** [reply]
The spine interface should allow to install Ninja from the database. Once installed, i.e. in the library, Ninja will appear in the launcher options with a checkbox to tick. 

If it does not appear in the launcher options or the checkbox is not tickable, you may need to refer to spine for help. 

To confirm, if Ninja is active, you can follow the instructions from the troubleshooting page in the Ninja wiki.

**Reply by justtoask** [reply]
I installed ninja in spine and can tick it, but it doesn't do anything, when I type `Ninja` into the console it says `Unknown command: NINJA`.

**Reply by szapp** [reply]
Steam applies an update by default that has to be disabled. I think you can find it in the properties of the game in the tab “betas” and setting it to “original”. 

Alternatively, you can remove the checkmark “start with Steam” in the Spine path settings. 

For details, please refer to the Spine support.
